src/config.c

```diff
@@ -2561,6 +2561,7 @@ standardConfig configs[] = {
     createBoolConfig("disable-thp", NULL, MODIFIABLE_CONFIG, server.disable_thp, 1, NULL, NULL),
     createBoolConfig("cluster-allow-replica-migration", NULL, MODIFIABLE_CONFIG, server.cluster_allow_replica_migration, 1, NULL, NULL),
     createBoolConfig("replica-announced", NULL, MODIFIABLE_CONFIG, server.replica_announced, 1, NULL, NULL),
+    createBoolConfig("aof-enable-auto-gc", NULL, MODIFIABLE_CONFIG, server.aof_enabled_auto_gc, 1, NULL, NULL),
```

Comment:
these new config interfaces need to be documented in redis.conf, and also listed in the top comment of the PR under some "interface changes section"
note that we use the PR top comment as a squash-merge commit comment as also for the purpose of release notes.